### ISSUE TITLE
Feature Request: Ability to embed logo and favicon directly into html via Base64 encoding.

### ISSUE THREAD

**Issue by zleonov** [anvilco/spectaql#807]
It would be useful if SpectaQL could encode the logo and favicon directly in the resulting html file. The general use case is the same as the `oneFile` config option.

Here is an example html:

```
<!DOCTYPE html>
<html>
  <body>
     <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12P4//8/w38GIAXDIBKE0DHxgljNBAAO9TXL0Y4OHwAAAABJRU5ErkJggg==" alt="sample red dot" />
  </body>
</html>
```

TL;DR:

In my specific case the generated documentation is deployed with our product on an application server. We have a very strict policy when it comes to exposing resources to the web. This requires putting images in one location, putting html files in a different location, configuring various settings, and so forth. To make it work we must manipulate the image paths in the resulting html file before deployment to match the location of the images on the application server.

While this can be automated, essentially this means we have to maintain two copies of the documentation. The original output, which we preview as we improve our documentation, and the output which gets deployed. Etc. Etc.

If there was a single file with everything embedded that would simply all these use cases.

**Comment by newhouse** [comment]
Feels reasonable to me

**Comment by BadgerBadgerBadgerBadger** [comment]
To add to this. I have a similar usecase where providing a URL instead of a file path would solve my issue.

**Comment by newhouse** [comment]
Support for this is coming today @BadgerBadgerBadgerBadger @zleonov 

You will be to supply a `embedLogo` and `embedFavicon` booleans to your config. If `true` and logo/favicon file was provided, it will be embedded as Base64 rather than loaded as a local resource from the `images/` directory.

You can also provide a `logoUrl`/`faviconUrl`, which will take precedent over the above, and be used for the src/href, if provided.

**Comment by newhouse** [comment]
This is published in `main` and on NPM in v`2.3.0`.

https://github.com/anvilco/spectaql/blob/main/config-example.yml#L27

https://github.com/anvilco/spectaql/blob/main/config-example.yml#L47

**Comment by zleonov** [comment]
This is great!
